modules/candidate_list/test/candidate_listTest.php

```diff
@@ -77,56 +90,212 @@ function testCandidateListPageLoads()
      */
     function testCandidateListAdvancedOptionsAppear()
     {
-
         $this->safeGet($this->url . "/candidate_list/");
         $bodyText = $this->webDriver
             ->findElement(WebDriverBy::cssSelector("body"))->getText();
         $this->assertContains("Access Profile", $bodyText);
-
         // Switch to Advanced mode
         $basicButton = $this->webDriver->findElement(WebDriverBy::Name("advanced"));
         $basicButton->click();
-
         // Go through each element and ensure it's on the page after clicking
         // advanced
         $scanDoneOptions = $this->webDriver->findElement(
             WebDriverBy::Name("scan_done")
         );
         $this->assertEquals("select", $scanDoneOptions->getTagName());
-
         $participantsStatusOptions = $this->webDriver->findElement(
             WebDriverBy::Name("Participant_Status")
         );
         $this->assertEquals("select", $participantsStatusOptions->getTagName());
-
         $dobOptions = $this->webDriver->findElement(WebDriverBy::Name("dob"));
         $this->assertEquals("input", $dobOptions->getTagName());
         // Not currently done
         //$this->assertEquals("date",$dobOptions->getAttribute("type"));
-
         $genderOptions = $this->webDriver->findElement(WebDriverBy::Name("gender"));
         $this->assertEquals("select", $genderOptions->getTagName());
-
         $numVisits = $this->webDriver->findElement(WebDriverBy::Name("Visit_Count"));
         $this->assertEquals("input", $dobOptions->getTagName());
         // Not currently done in Loris.
         //$this->assertEquals("number",$dobOptions->getAttribute("type"));
         //$this->assertEquals("0",$dobOptions->getAttribute("min"));
-
         $edcOptions = $this->webDriver->findElement(WebDriverBy::Name("edc"));
         $this->assertEquals("input", $edcOptions->getTagName());
         // Not currently done
         //$this->assertEquals("date",$edcOptions->getAttribute("type"));
-
         $latestVisitOptions = $this->webDriver->findElement(
             WebDriverBy::Name("Latest_Visit_Status")
         );
         $this->assertEquals("select", $latestVisitOptions->getTagName());
-
         $feedbackOptions = $this->webDriver->findElement(
             WebDriverBy::Name("Feedback")
         );
         $this->assertEquals("select", $feedbackOptions->getTagName());
     }
+    /**
+     * Performs various searches by PSCID (and PSCID only).
+     *
+     * @return void.
+     */
+    function testFilterByPscid()
+    {
```

Comment:
// Stop here and mark this test as incomplete.
$this->markTestIncomplete(
'This test is using a React component'
);